config/uiowa.edu/block.block.socialmedia.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 0e4eea1e-87e4-455b-8ba7-8143904753ee
+langcode: en
+status: true
+dependencies:
+  config:
+    - system.menu.social
+  module:
+    - menu_block
+  theme:
+    - uids_base
+id: socialmedia
```

Comment:
One note: We have been using a somewhat informal custom of naming blocks something like `uids_base_socialmedia` to prevent conflicts with other themes. I'm definitely open to more discussion on this topic, just wanted to point that out.

Reply:
ah, i didn't know the block machine name could be renamed. adjusted to match your example.